File: solutions/RepeatStringEasy/solve/cpp/RepeatStringEasySolver.cpp
#include <fstream>
#include <topcoder>
#include "RepeatStringEasy.cpp"
namespace tc = TopCoder;

int main(int argc, char const *argv[]) {
  try {
    std::ifstream ifs(argv[1]);
    string s; tc::read(ifs, s);
    ifs.close();

    std::ofstream ofs(argv[2]);
    RepeatStringEasy solver;
    tc::show(ofs, solver.maximalLength(s));
    ofs.close();
  } catch (std::exception &e) {
    std::cerr << e.what() << std::endl;
  }
  return 0;
}

File: solutions/RepeatStringEasy/solve/java/RepeatStringEasySolver.java
import java.io.FileReader;
import java.io.FileWriter;

import java.util.List;
import java.util.ArrayList;

import org.topcoder.TopcoderReader;
import org.topcoder.TopcoderWriter;
import org.topcoder.TypeRef;

public class RepeatStringEasySolver {
    public static void main(String[] args) {
    try {
        TopcoderReader reader = new TopcoderReader(new FileReader(args[0]));
        String s = (String) reader.next(String.class);
        reader.close();

        RepeatStringEasy solver = new RepeatStringEasy();
        TopcoderWriter writer = new TopcoderWriter(new FileWriter(args[1]));
        writer.write(solver.maximalLength(s));
        writer.close();
    } catch (Exception err) {
        err.printStackTrace(System.err);
    }
    }
}
